src/status_im/data_store/realm/schemas/account/v22/core.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im.data-store.realm.schemas.account.v22.core
```

Comment:
why did you create a v22 in your rebase ? we only do one migration per version

Reply:
I already had to do the rebase on my branch that is based of yours https://github.com/status-im/status-react/blob/88fa360b5f26cdfa306bbe6ad0eaa9327d8e598a/src/status_im/data_store/realm/schemas/account/v21/core.cljs